### PR TITLE
Feature/general styles setup

### PR DESCRIPTION
**Pull request by ArleyAV**
Set colors within variables
Set breakpoints
Set grid layout for mobile (No provided desktop grid yet)
Set text styles
Set some rules for global reset

### CODE REVIEW

**Comment by jsebax** [comment]
@ArleyAV You don't need to have the local files for the fonts (eot, ttf, woff, etc), because you are downloading them using `Google Fonts` url.

**Comment by ArleyAV** [comment]
@jsebax Font files have been deleted!

**Review by juan-ortizc — APPROVED** [review]

File: src/scss/abstracts/_mixins.scss [hunk 1]
```diff
@@ -1 +1,10 @@
 /* Sass Mixins */
+
+//Define the website's breakpoints
+@mixin viewport-desktop {
+  @media (min-width: 728px) {
+    @content;
+  }
+}
+
+
```
Comment (camigomezdev) [inline src/scss/abstracts/_mixins.scss]: please delete the extra line

File: src/scss/base/_reset.scss [hunk 2]
```diff
@@ -1 +1,15 @@
...
+  margin: 0;
+  list-style: none;
+}
+
+img,
+video {
+  width: 100%
+}
+
+
```
Comment (camigomezdev) [inline src/scss/base/_reset.scss]: please delete the extra line

File: src/scss/base/_typography.scss [hunk 3]
```diff
@@ -1 +1,61 @@
...
+  &-normal {
+    font-size: 16px;
+    font-weight: 400;
+    letter-spacing: -0.2px;
+    line-height: 1.1;
+    &--bold {
+      font-weight: 700;
+    }
+  }
+}
```
Comment (camigomezdev) [inline src/scss/base/_typography.scss:61]: Please add an empty line here

File: src/scss/base/_grid.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+//Grid Mobile
+.grid{
+  display: grid;
+  column-gap: 5px;
+  grid-template-columns: repeat(12, 1fr);
+  max-width: 360px;
+  width: 100%;
+  //Grid Desktop (No defined yet)
+}
```
Comment (camigomezdev) [inline src/scss/base/_grid.scss:10]: Please add an extra line please

File: src/scss/abstracts/_variables.scss [hunk 5]
```diff
@@ -1 +1,13 @@
...
+$black: #000000;
+$blue-grey: #474f78;
+$blue-grey-dark: #323755;
+$blue-grey-light: #adb0c4;
+$green: #0c5a45;
+$green-dark: #043326;
+$grey: #d8d8d8;
+$orange: #ffcc36;
+$yellow: #ffcc36;
+$white: #ffffff;
```
Comment (camigomezdev) [inline src/scss/abstracts/_variables.scss:13]: Here

**Review by camigomezdev — APPROVED** [review]

**Review by jsebax — APPROVED** [review]

**Pull request merged**